pkgs/applications/virtualization/docker/default.nix

```diff
@@ -32,7 +32,7 @@ rec {
       patches = [];
     });
 
-    docker-containerd = containerd.overrideAttrs (oldAttrs: {
+    docker-containerd = (callPackage ./containerd.nix {}).overrideAttrs (oldAttrs: {
```

Comment:
I'd add a note here as well that this should switch back to overriding `pkgs.containerd`.

Reply:
If you rebase on master this isn't needed now, we can keep the `pkgs.containerd` override and drop the pinned `buildGoPackage` version.